utils/refactor-check-compiles.py

```diff
@@ -88,12 +88,15 @@ def main():
     with open(temp_file_path, 'wb') as f:
         f.write(dump_rewritten_output)
 
+    extra_frontend_args = []
+    if '-enable-experimental-concurrency' in unknown_args:
+        extra_frontend_args.append('-enable-experimental-concurrency')
```

Comment:
I’d rather add this to the known args above instead of scraping the flag from `unknown_args`. At least that way it’s documented which flags will be forwarded to `swift-frontend`.

Reply:
Updated - it's now a flag.